File: ConsoleApplication1/UnitTest1/unittest1.cpp
#include "stdafx.h"
#include "CppUnitTest.h"
#include "../ConsoleApplication1/dc.h"


using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace UnitTest1
{		
	TEST_CLASS(UnitTest1)
	{
	public:
		
		TEST_METHOD(TestMethod1)
		{
			
			// TODO: 在此输入测试代码
			Assert::AreEqual(110, abc(5, 6));

		}

	};
}

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs
using WindowsFormsApp1;
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace WindowsFormsApp1.Tests
{
    [TestClass()]
    public class UnitTest1
    {
        [TestMethod()]
        public void aaTest()
        {
            Assert.Fail();
        }
    }
}

namespace UnitTestProject1
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
        }
        [TestMethod]
        public void TestMethod2()
        {
            WindowsFormsApp1.Form1 abc = new Form1();

            //Assert.Fail();
            Assert.AreEqual(12, abc.aa(10, 2));
        }
    }
}
